.github/workflows/scripts/addClosingMessage.js

```diff
@@ -0,0 +1,75 @@
+// @ts-check
+
+/**
+ * @param {Object} params
+ * @param {import("@actions/core")} params.core
+ * @param {ReturnType<import("@actions/github").getOctokit>} params.github
+ * @param {import("@actions/github").context} params.context
+ */
+module.exports = async ({ core, context, github }) => {
+  try {
+    const owner = context.repo.owner;
+    const repo = context.repo.repo;
+    const issueNumber = context.issue.number;
+
+    const issue = await github.rest.issues.get({
+      owner,
+      repo,
+      issue_number: issueNumber,
+    });
+
+    const repositoryMap = {
+      'mui-x': 'x',
+      'material-ui': 'core',
```

Comment:
```suggestion
      'material-ui': 'material-ui',
```

The notion of core is progressively going away. We have standalone libraries and integrated ones, e.g. Shadcn = Tailwind CSS + Radix Primitives.